### PR TITLE
meshca: Pull in the MeshCA config proto from grpc-proto repo.

### PR DESCRIPTION
**Pull request by easwars**

### CODE REVIEW

**Comment by easwars** [comment]
I have excluded the `pb.go` files from the vet check which was failing earlier. And when importing into g3, we can exclude these `pb.go` files and use the canonical versions instead. PTAL.

File: regenerate.sh [hunk 1]
```diff
@@ -39,6 +39,19 @@ mkdir -p ${WORKDIR}/googleapis/google/rpc
...
 curl --silent https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto > ${WORKDIR}/googleapis/google/rpc/code.proto
 
+# Pull in the following repos to build the MeshCA config proto.
+ENVOY_API_REPOS=(
+  "https://github.com/envoyproxy/data-plane-api"
+  "https://github.com/cncf/udpa"
+  "https://github.com/envoyproxy/protoc-gen-validate"
+)
+for repo in ${ENVOY_API_REPOS[@]}; do
+  dirname=`basename ${repo}`
```
Comment (dfawley) [inline regenerate.sh:49]: Use `$(` instead of backticks please.

File: regenerate.sh [hunk 2]
```diff
@@ -52,15 +65,30 @@ SOURCES=(
...
+# one of the sources.
+MAPPINGS=(
+  "grpc/service_config/service_config.proto=/internal/proto/grpc_service_config"
+  "envoy/config/core/v3/config_source.proto=github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+)
+OPTS=""
+for i in "${MAPPINGS[@]}"
+do
+  OPTS+="M$i,"
+done
```
Comment (dfawley) [inline regenerate.sh:82]: I don't think this extra complexity is a good idea.  Just write it literally:

```
OPTS=Mgrpc/service_config/service_config.proto=/internal/proto/grpc_service_config,\
Menvoy/config/core/v3/config_source.proto=github.com/envoyproxy/go-control-plane/envoy/config/core/v3
```

(Also, the code above leaves a trailing comma which apparently works now, but doesn't seem like a good thing to be doing.)

File: regenerate.sh (hunk 1 above)
Comment (easwars) [reply regenerate.sh]: Done.

File: regenerate.sh (hunk 2 above)
Comment (easwars) [reply regenerate.sh]: Done.

**Pull request merged**